components/customForm.vue

```diff
@@ -1,397 +1,424 @@
-        font-family: inherit;
-        font-weight: 700;
-        cursor: pointer;
-        width: 36px;
-        min-width: 36px;
-        height: 32px;
-        -webkit-box-pack: center;
-        justify-content: center;
-        background-color: rgb(79, 169, 179);
-        color: rgb(1, 2, 2);
-        transition: background-color 0.2s ease 0s, color 0.2s ease 0s, border-color 0.2s ease 0s, opacity 0.2s ease 0s;
-        outline: none;
-        margin: 0;
-        padding: 0;
-        border-width: 1px;
-        border-style: solid;
-        border-image: initial;
-        border-color: transparent;
-        border-radius: 4px 0 0 4px;
-    }
-
-    .btn-navigate:hover, .btn-navigate:disabled {
-        background-color: rgb(105, 182, 190);
-    }
-
-    .btn-navigate:disabled {
-        background-color: rgba(105, 182, 190, 0.3);
-    }
-
-    .btn-navigate:last-of-type {
-        border-radius: 0 4px 4px 0;
-        transform: translateX(-5px);
-        border-left: 1px solid grey;
-    }
-
-    .form-button, .form-button:hover, .form-button:focus {
-        display: inline-block;
-        margin: 1rem;
-        position: relative;
-        font-family: inherit;
-        font-weight: 700;
-        cursor: pointer;
-        box-shadow: rgba(0, 0, 0, 0.1) 0 3px 12px 0;
-        min-height: 40px;
-        background-color: rgb(79, 169, 179);
-        color: rgb(1, 2, 2);
-        transition: background-color 0.2s ease 0s, color 0.2s ease 0s, border-color 0.2s ease 0s, opacity 0.2s ease 0s;
-        outline: none;
-        border-width: 1px;
-        border-style: solid;
-        border-image: initial;
-        padding: 6px 14px;
-        border-color: transparent;
-        border-radius: 4px;
-    }
-
-    .form-button.accept-button, .form-button.accept-button:hover, .form-button.accept-button:focus {
-        display: block;
-        margin: 1rem 0;
-    }
-
-    .navigation-wrapper {
-        text-align: right;
-        padding: 1.5rem;
-    }
-
-    .form-control::placeholder {
-        color: lightgray;
-        font-style: italic;
-    }
-
-    .error {
-        color: red;
-    }
-
-</style>
+<template>
+  <div class="form-wrapper">
+    <transition-group name="fade">
+      <div
+        v-if="step !== -1 && !displayCaptcha"
+        :key="'form-wrapper'"
+        :class="{'submitted': submittedField}"
+      >
+        <h5 class="step-title" :key="'text' + step">{{ stepProps.text }}</h5>
+        <h6 class="step-subtitle" v-if="stepProps.subtitle" :key="'subtitle' + step">{{ stepProps.subtitle }}</h6>
+
+        <div class="error" v-if="stepProps.error">{{ stepProps.error }}</div>
+
+        <b-button
+          :key="'button' + step + option.text"
+          class="form-button"
+          v-for="option in getInputOptions()"
+          @click="storeOptionInfo(option)"
+        >{{ option.text }}</b-button>
+
+        <img class="step-image" v-if="stepProps.image" :src="stepProps.image"/>
+
+        <b-form-select :ref="stepProps.id"

```

Comment:
This part is still open, to create multiselect options as it wasn't existing a similar input on the rescueapp one, for now they are all working as normal text but once working it will be a matter of changing the input type on the jsons